tasks.py

```diff
@@ -138,9 +143,3 @@ def docserve(c):
     server.watch("CONTRIBUTING.rst", lambda: docbuild(c))
     server.watch("docs/*.rst", lambda: docbuild(c))
     server.serve(port=DOCS_PORT, root="_build/html")
-
```

Comment:
this must be restored

Reply:
You're right, it was lost on copy/paste probably